pkg/login/social/common.go

```diff
@@ -57,3 +67,30 @@ func HttpGet(client *http.Client, url string) (response HttpGetResponse, err err
 	err = nil
 	return
 }
+
+func (s *SocialBase) searchJSONForAttr(attributePath string, data []byte) string {
+	if attributePath == "" {
+		s.log.Error("No attribute path specified")
```

Comment:
Are we sure we shouldn't actually return an error here? This is in effect swallowing the error. cc: @marefr